lab-nodemailer/routes/auth.js

```diff
@@ -0,0 +1,121 @@
+const express = require('express')
+const passport = require('passport')
+const router = express.Router()
+const User = require('../models/User')
+const nodemailer = require('nodemailer')
+
+// Bcrypt to encrypt passwords
+const bcrypt = require('bcrypt')
+const bcryptSalt = 10
+
+const transporter = nodemailer.createTransport({
+  service: 'Gmail',
+  auth: {
+    user: process.env.MAIL,
+    pass: process.env.MAIL_PWD,
+  },
+})
+
+router.get('/login', (req, res, next) => {
+  res.render('auth/login', { message: req.flash('error') })
+})
+
+router.post(
+  '/login',
+  passport.authenticate('local', {
+    successRedirect: '/',
+    failureRedirect: '/auth/login',
+    failureFlash: true,
+    passReqToCallback: true,
+  })
+)
```

Comment:
Asi es como pueden agregar un controller para la respuesta del login existoso con un redireccionamiento custom. 👍 

```suggestion
router.post("/login", passport.authenticate("local"), (req, res) => {
  res.redirect(`/auth/profile/${req.user.id}`);
});
```